### ISSUE TITLE
Icons and logos don't reset when changing themes

### ISSUE THREAD

**Issue by novamaru** [spruceUI/spruceOS#88]
I noticed when changing themes in v2.3 between the three MinUInspired options that the logos vs icons may not reset if you select the `MinUInspired (Grey)` option. I'm not entirely sure which combination is supposed to be default, but I think if you start with `MinUInspired (Grey)`, you get the same icons as the `MinUInspired (Icons)` theme, but in all white without the purple highlighting. But if you change the theme to `MinUInspired (logos)` and then back to `MinUInspired (Grey)`, you get the white/grey color but now with logos instead of icons. And I prefer this resulting combination, but it seems like it might be a minor bug in the initialization of each theme.

**Comment by spruceUI** [comment]
MinUInspired (Grey) uses the logos and not the icons. I have tested this switching a number of times now and cannot get it to do anything except what it is supposed to do.

If you want to have Icons instead you need to manually adjust the Theme folder

Feel free to DM me on Discord if you need help adjusting the icons for the themes.

-Sundownersport

**Comment by novamaru** [comment]
Thanks for looking at it. I prefer the logos to the icons. Not sure if this helps, but what I noticed might have been right after an initial installation. I have two A30s, and I upgraded the first one from Spruce version 1.2.4b to 2.3, and then I was playing around with the new themes and settled on the MinUI Grey. Then I updated the second A30, which was brand new, so here I wiped the card completely and did a fresh install of Spruce 2.3. And I *think* it was here that I noticed that going straight to MinUI Grey somehow changed the color but not icons. So I changed the theme to MiniUI Logos and back to Grey and then it looked right. Unfortunately, because of this, I'm not 100% of when I saw it or exactly what order of operations I used. But I'm pretty sure I ended up with grey plus icons at one point, because that really jumped out at me after having tried out all of the options. Anyway, either it's nothing or it's very minor.
